README.markdown

```diff
@@ -446,9 +446,11 @@ Generated code will be placed in the Gradle build directory.
 
 - With `--ts_proto_opt=outputServices=false`, or `=none`, ts-proto will output NO service definitions.
 
-- With `--ts_proto_opt=outputBeforeRequest=true`, ts-proto will add a function definition to the Rpc interface definition with the signature: `beforeRequest(request: <RequestType>)`. It will will also automatically set `outputTypeRegistry=true` and `outputServices=true`. Each of the Service's methods will call `beforeRequest` before performing it's request.
+- With `--ts_proto_opt=outputBeforeRequest=true`, ts-proto will add a function definition to the Rpc interface definition with the signature: `beforeRequest(service: string, message: string, request: <RequestType>)`. It will will also automatically set `outputServices=default`. Each of the Service's methods will call `beforeRequest` before performing it's request.
```

Comment:
@lukealvoeiro This is admittedly a little late, since we've already released these `outputBefore/After` flags, but wdyt of renaming all of these to:

* `rpcBeforeRequest=true`
* `rpcAfterResponse=true`
* `rpcErrorHandler=true`

The disclaimer/mea culpa is that I've not been great at having a super-consistent naming convention for flags, we just kind of make them up as we go, but given these are all related to the `Rpc` interface, that seems like a good prefix?

Reply:
No worries, those names are definitely better. Thanks for the quick 👀 on this PR :)